draco/analysis/transform.py

```diff
@@ -432,7 +432,18 @@ class MModeTransform(task.SingleTask):
     """Transform a sidereal stream to m-modes.
 
     Currently ignores any noise weighting.
+
+    Attributes
+    ----------
+    set_telmmax : bool
+        Whether to slice the data at the maximum m the telescope can resolve
+        after fourier transforming. Default : false.
     """
+    set_telmmax = config.Property(proptype=bool, default=False)
+
+    def setup(self, manager):
```

Comment:
This introduces a mandatory dependency on the `manager` but that's actually unnecessary if you don't set `set_telmmax`.

I would change this to make manager optional `i.e. manager=None`, and then if `manager` is not None this is equivalent to setting `set_telmmax`. Then I'd remove the `set_telmmax` option entirely.

Reply:
Ok that makes sense. I made the changes and just going to quickly test the code.